modules/ocf_irc/manifests/ircd.pp

```diff
@@ -0,0 +1,32 @@
+class ocf_irc::ircd {
+  package { 'inspircd':; }
+
+  service { 'inspircd':
+    ensure  => 'running',
+    restart => 'service inspircd reload',
+    require => Package['inspircd'],
+  }
```

Comment:
No, it's fine without it, I just thought it would be best if it's running. I'll remove that then.

I've changed restart so that it reloads gracefully when any config files are changed instead of performing a full restart and kicking everyone off the server.